package.json

```diff
@@ -42,7 +42,7 @@
     "typescript": "^4.7.4"
   },
   "peerDependencies": {
-    "@sanity/client": "^3.0.0",
+    "@sanity/client": "^5.2.2",
```

Comment:
I think you should loosen this dependency version to `^5.0.0` since that's the version that introduced the breaking change / new type.

Reply:
Yeah if you could change that we can get this PR integrated. I'll try to make a release somewhere this week.

Reply:
@cbetancourt That's a fair shout - I've updated the @sanity/client version to `^5.0.0`.